EIPS/eip-sc.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: <to be assigned>
+title: Interface for Security Contact of Smart Contract
+description: An interface for security notice using asymmetric encryption
+author: Zainan Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/erc-interface-for-security-contract/10303
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-09
+requires: 165
+---
+
+An interface for smart contract developer to specify to communicate to them in secured way.
+A typical use case is for security researcher to report to developer the
+security vulnerability.
+
+## Abstract
+An interface for security notice using asymmetric encryption. The interface exposes a asymmetric encryption key
+and a destination of delivery.
+
+## Motivation
+When security researchers identify security risk of a smart contract, currently there is no official way to
+report the problem to the smart contract maintainer, such as MakerDAO's issue.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+```solidity
+interface ERC_Security_Contact {
+
+    /// REQUIRED
+    function getSecurityContact(bytes memory data) public view
+    returns (
+        uint8 type,
+        bytes memory publicKey,
+        bytes memory extraData
+    );
+
+    /// OPTIONAL
```

Comment:
I'd recommend either splitting these optional functions into their own interface(s), or defining what they return when not implemented.

Reply:
Sounds good, would you be ok if I leave it to a follow up revision but to get this file check in first